src/main.py

```diff
@@ -247,12 +247,11 @@ def mrl_fix(url):
 
 
 def internet_on():
-	logger.info("Checking Internet Connection...")
 	try:
 		requests.get('https://api.amazon.com/auth/o2/token')
 		logger.info("Connection OK")
 		return True
-	except:  # pylint: disable=bare-except
+	except requests.exceptions.RequestException:
 		logger.error("Connection Failed")
```

Comment:
Would be good to say what exactly happened here, so it will be easier for end users to debug and they won't have to open support issue.